composer.json

```diff
@@ -19,7 +19,7 @@
 		"fakerphp/faker": "^1.9",
 		"mikey179/vfsstream": "^1.6",
 		"phpstan/phpstan": "^0.12",
-		"phpunit/phpunit": "^8.5",
+		"phpunit/phpunit": "^9.5",
```

Comment:
probably change to `~8.5.0` for `dev-develop`

Reply:
```suggestion
 		"phpunit/phpunit": "~8.5.0", 
```

Reply:
Since 8.5 uses phpunit/php-token-stream, which has been marked abandoned, I would vote for upgrading to 9.5 nevertheless, or does this introduce issues?

Reply:
that live in 4.1 branch https://github.com/codeigniter4/CodeIgniter4/blob/4.1/composer.json#L22 . 
4.0.x tag still support php 7.2

Reply:
PHPUnit 9 requires PHP 7.3. CI4 requires PHP 7.2 or later.

Reply:
The goal will be for `4.0.5` to support PHP 7.2 - 8.0, which will require PHPUnit `^8.5 || ^9.0`. It remains to be seen if this will be possible with our current workload. #4012 should cover this change.

Reply:
Ok, it can be closed then.